modules/migration-configuring-mcg.adoc

```diff
@@ -11,30 +11,38 @@
 = Retrieving Multicloud Object Gateway credentials
 
 ifdef::installing-3-4,installing-mtc[]
-You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint in order to configure MCG as a replication repository for the {mtc-full} ({mtc-short}).
+You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint to configure MCG as a replication repository for the {mtc-full} ({mtc-short})
```

Comment:
I might leave _in order_ here.  It makes it more clear.  Now you have to stop for a moment to think:

 _endpoint to configure_  Are you retrieving the endpoint that you want to configure or retrieving the endpoint so that you can use it to configure something else?

Reply:
@RichardHoch I just looked in the IBM Style Guide. It suggested using _to_ over _in order to_ "in most cases" to avoid wordiness. Based on what stillalearner replies (I would think the former is right), how about:

```suggestion
You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint, which you need to configure MCG as a replication repository for the {mtc-full} ({mtc-short})
```

Reply:
@mburke5678 A fine solution, pending QA's answer. :)

Reply:
@mburke5678  QA confirmed my reading of the discussion of the S3 endpoint. Made the change you suggested.